dev-python/pyzbar/pyzbar-0.1.5.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+inherit distutils-r1
+
+DESCRIPTION="Read one-dimensional barcodes and QR codes from Python."
+HOMEPAGE="https://github.com/NaturalHistoryMuseum/pyzbar/"
+SRC_URI="https://github.com/NaturalHistoryMuseum/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	media-gfx/zbar
+	dev-python/pillow
+	dev-python/numpy
+	python_targets_python2_7? (
```

Comment:
Get rid of the `python_targets_python2_7` conditional, and depend on `virtual/python-enum34` and `virtual/python-pathlib`.